amgut/lib/locale_data/british_gut.py

```diff
@@ -465,7 +465,9 @@
     'ADD_NEW': "Add a New Human Sample Source",
     'CONTINUE': "Continue",
     'EXPLANATION': "You have entered the add human source workflow. During this workflow you will add a human source that represents whoever is being sampled. You be asked for consent to join the project and then asked survey questions.",
-    'ONCE_ADDED': "Once you have added a human source, you will then see the name of that source in the left menu, and you will also have an option for adding a sample to that source. When you click that, you will be able to select the appropriate barcode and add sample metadata."
+    'ONCE_ADDED': "Once you have added a human source, you will then see the name of that source in the left menu, and you will also have an option for adding a sample to that source. When you click that, you will be able to select the appropriate barcode and add sample metadata.",
+    'ELECTRONIC_SIGNATURE': 'In order to participate in this study, you will need to sign a research consent form. This can be done electronically. To consent to using an electronic signature, please click the button below.',
+    'ELECTRIC_SIG_CONSENT': 'I consent to using an electronic signature'
```

Comment:
We probably don't want to delay, but if it's quick, is it worth running this new text by a "native british speaker"?